src/abc/code/Classes/AbcModule.php

```diff
@@ -16,7 +16,9 @@ public static function load($name) {
 				Requirements::javascript(ABC_PATH."/javascript/library/jQuery/jquery-1.10.2.min.js");
 				LeftAndMainHelper::require_unblock(THIRDPARTY_DIR."/jquery/jquery.js");
 				LeftAndMainHelper::require_block(ABC_PATH."/javascript/library/jQuery/jquery-1.10.2.min.js");
-				LeftAndMain::require_javascript(THIRDPARTY_DIR."/jquery/jquery.js");
+				// @todo: this call was here pre ss4, that function is no longer there.
```

Comment:
I'm unsure how much of the custom requirements stuff is still needed really, used to be that if you used requirements::javascript that would be loaded in the CMS. The point of this stuff was to provide more granular controls about where stuff was loaded, we don't really use Requirements anymore outside of the CMS so it's really of limited use, also the jq version is stupid old and should be updated to a supported version